site/en/docs/extensions/reference/webRequest/index.md

```diff
@@ -27,6 +27,8 @@ As of Chrome 108, you can asynchronously supply credentials for [`onAuthRequired
 events](#event-onAuthRequired) if you use the `"webRequest"` and `"webRequestAuthProvider"`
 permissions.
 
+**note** As of Manifest V3, the `webRequestBlocking` manifest key is replaced by `declarativeNetRequest`, which enables use of the [declarativeNetRequest API](/docs/extensions/reference/declarativeNetRequest/).
```

Comment:
A couple of thoughts:

- Policy installed extensions can still use `webRequestBlocking` (see the paragraph two above). Should we merge those two paragraphs to make it clear there are multiple options?
- Should we use an [aside](https://developer.chrome.com/docs/handbook/components/#asides) to format this?

Reply:
The answer to the last question is always 'yes'. Part of our job requirements is to implement corporate styles consistently across our site or sites.